src/Components/Admin/Admin.tsx

```diff
@@ -111,10 +132,18 @@ class Admin extends React.Component<any, any> {
       .firestore()
       .collection("Workshop")
       .onSnapshot((snapshot) => {
-        let arr = [];
+        let arr: workshop[] = [];
         //save each workshop into an array
-        snapshot.forEach((snap) => {
-          arr.push(snap.data());
+        snapshot.forEach((snap: workshopFirebase) => {
+          let workshopObject = {
```

Comment:
Explicit definition instead of type inference
```suggestion
          let workshopObject: workshop = {
```